moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -88,6 +85,15 @@ geometry_msgs::msg::TransformStamped convertIsometryToTransform(const Eigen::Iso
 
   return output;
 }
+
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_servo.servo_calcs");
+constexpr auto ROS_LOG_THROTTLE_PERIOD = std::chrono::milliseconds(3000).count();
+static constexpr double STOPPED_VELOCITY_EPS = 1e-4;  // rad/s
+
+// This value is used when configuring the main loop to use SCHED_FIFO scheduling
+// We use a midpoint RT priority to allow maximum flexibility to users
+// Reference: https://man7.org/linux/man-pages/man2/sched_setparam.2.html
+int const THREAD_PRIORITY = 50;
```

Comment:
Suggest picking a lower number than the one we chose for ros2_control to avoid higher variance in the jitter that will be caused by the schedule randomly choosing this or controller_manager's thread to run if both are runnable. I would probably choose 40 to have lower number but leave some numbers between this and controller_manger.

Reply:
done